src/phases/query/QueryStatsStages.tmpl

```diff
@@ -0,0 +1,161 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of helper stages for QueryStats workloads. Each stage represents a single
```

Comment:
This sentence is not really making sense... I think it will make sense once I read further, but ideally it makes sense _before_ I read further.

Questions I have when reading this:
* When you say "each stage" - there are 2.5 here in this file? The SetupStage and the QueryStatsStageTemplate. What is this mysetrious "StageTemplate"? Do we need it to be a separate thing?
* Related to my other question but what is "a single workload variant"? A "such as" might help here.
* Stages can be grouped together ... why and how?